public/app/features/integrated-alerting/components/Table/Table.tsx

```diff
@@ -4,6 +4,7 @@ import { css } from 'emotion';
 import { Spinner, useStyles } from '@grafana/ui';
 import { getStyles } from './Table.styles';
 import { TableProps } from './Table.types';
+import { EmptyBlock } from '../EmptyBlock/EmptyBlock';
```

Comment:
Normally, we try to use an index.ts file to avoid importing components directly.
I.e. `import { EmptyBlock } from '../EmptyBlock';`

Reply:
Done 👌